SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -386,8 +386,13 @@ private void InstanceTrackError(System.Exception exception, IDictionary<string,
                 }
                 properties = PropertyValidator.ValidateProperties(properties, "HandledError");
                 var exceptionAndBinaries = ErrorLogHelper.CreateModelExceptionAndBinaries(exception);
-                var log = new HandledErrorLog(exception: exceptionAndBinaries.Exception, binaries:exceptionAndBinaries.Binaries, properties: properties, id: Guid.NewGuid(), device: null, userId: UserIdContext.Instance.UserId); ;
+                var errorId = Guid.NewGuid();
+                var log = new HandledErrorLog(exception: exceptionAndBinaries.Exception, binaries:exceptionAndBinaries.Binaries, properties: properties, id: errorId, device: null, userId: UserIdContext.Instance.UserId);
                 Channel.EnqueueAsync(log);
+                if (attachments?.Length > 0)
+                {
+                    SendErrorAttachmentsAsync(errorId, attachments);
```

Comment:
Are we checking length in both places? Can we move the check inside?

Reply:
@guperrot yep, don't need the check. It is handled by the foreach in the function.